CONTRIBUTING.md

```diff
@@ -57,6 +57,17 @@ After running for a while, the command should print
 ```
 ALL TESTS PASSED
 ```
+## Vendoring
+
+We keep a cached copy of dependencies under the vendor/ directory,
+managing updates via [govendor](http://github.com/kardianos/govendor).
+
+Pull requests should only include vendor/ updates if they are
+part of the same change, be it a bugfix or a feature addition.
+
+In any case, the vendor/ update needs to be justified as part
```

Comment:
I meant get rid of `In any case, `, but that might've been ambiguous :slightly_smiling_face: